error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/NonStaticImport.java

```diff
@@ -61,6 +61,8 @@ public final class NonStaticImport extends BugChecker implements CompilationUnit
       ImmutableSet.of(
           "com.google.common.base.Strings",
           "com.google.errorprone.BugCheckerRefactoringTestHelper.TestMode",
+          "com.google.errorprone.util.ASTHelpers",
+          "com.google.errorprone.VisitorState",
```

Comment:
To match the GJF's import order:
```suggestion
          "com.google.errorprone.VisitorState",
          "com.google.errorprone.util.ASTHelpers",
```